roles/kubernetes-apps/csi_driver/cinder/defaults/main.yml

```diff
@@ -15,3 +15,4 @@ cinder_cacert: "{{ lookup('env','OS_CACERT') }}"
 # For now, only Cinder v3 is supported in Cinder CSI driver
 cinder_blockstorage_version: "v3"
 cinder_csi_controller_replicas: 1
+cinder_ignore_volume_az: false
```

Comment:
Why don't we name `cinder_csi_ignore_volume_az` like the above cinder_csi_controller_replicas?

Reply:
Because of `cinder_topology`, `cinder_blockstorage_version` and all the variables defined in roles/kubernetes-apps/csi_driver/cinder/defaults/main.yml follow this pattern.

All `cinder_` could be renamed to `cinder_csi`, it could be more meaningful. What do you think?

Reply:
It is fine to rename to `cinder_csi_ignore_volume_az` only in this pull request for keeping the scope small.
and it is nice to have another pull request for renaming the existing ones if these options also could make confusion, I think.

Reply:
Agreed. I changed it to `cinder_csi_ignore_volume_az`.